### DISCUSSION TITLE
Tying the audio levels to a specific application in volume mixer

### DISCUSSION THREAD

**Discussion by zacharygoodrich** [ShankarBUS/ModernFlyouts#913] [Ideas]
I tend to keep my system volume the same, but will change my spotify audio levels frequently in volume mixer through a hotkey. Since I have no clear way of seeing what volume level it is exactly at, I thought this app would be perfect to report that sort of thing... for me atleast.

**Comment by Samuel12321** [comment]
@zacharygoodrich what a coincidence, we thought the same thing. The team is working on this for the next update 0.11 which should be released this month.

**Reply by zacharygoodrich** [reply]
oooo thank you! im excited :)
